Source/WebKit/UIProcess/API/Cocoa/_WKWebExtensionTab.h

```diff
@@ -314,21 +295,13 @@ WK_API_AVAILABLE(macos(WK_MAC_TBA), ios(WK_IOS_TBA), visionos(WK_XROS_TBA)) WK_S
 
 /*!
  @abstract Called to reload the current page in the tab.
+ @param fromOrigin A boolean value indicating whether to reload the tab from the origin, bypassing the cache.
  @param context The context in which the web extension is running.
  @param completionHandler A block that must be called upon completion. It takes a single error argument,
  which should be provided if any errors occurred.
- @discussion Reloads the main web view via `reload` if not implemented.
- */
-- (void)reloadForWebExtensionContext:(_WKWebExtensionContext *)context completionHandler:(void (^)(NSError * _Nullable error))completionHandler;
-
-/*!
- @abstract Called to reload the current page in the tab, bypassing the cache.
- @param context The context in which the web extension is running.
- @param completionHandler A block that must be called upon completion. It takes a single error argument,
- which should be provided if any errors occurred.
- @discussion Reloads the main web view via `reloadFromOrigin` if not implemented.
+ @discussion Reloads the main web view via `reload` or `reloadFromOrigin` if not implemented.
  */
-- (void)reloadFromOriginForWebExtensionContext:(_WKWebExtensionContext *)context completionHandler:(void (^)(NSError * _Nullable error))completionHandler;
+- (void)reloadForWebExtensionContext:(_WKWebExtensionContext *)context fromOrigin:(BOOL)fromOrigin completionHandler:(void (^)(NSError * _Nullable error))completionHandler;
```

Comment:
Should this be:
`reloadFromOrigin:(BOOL)fromOrigin forWebExtensionContext:`?

For consistency with the other methods? I wasn't sure if API style meant the `_WKWebExtensionContext` should be the last argument. If not, as-is is fine.

Reply:
Hmm, good point.